flytekit/annotated/notification.py

```diff
@@ -0,0 +1,68 @@
+from typing import List
+
+from flytekit.models import common as _common_model
+from flytekit.models.core import execution as _execution_model
+
+
+class Notification(_common_model.Notification):
```

Comment:
Add a comment that this is duplicate because we don't want to deal with metaclass?

Reply:
done